docs/pbc/all/payment-service-provider/202410.0/base-shop/payment-method-strategies.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Payment method strategies
+description: This doc describes the different payment method strategies available in Spryker Commerce OS.
+last_updated: Nov 5, 2024
+template: howto-guide-template
+originalLink: ...
+originalArticleId: ...
+redirect_from:
+  - ...
+---
+
+
+## Payment Method Strategies
+
+Spryker and the provided Payment Service Provide Apps (PSP) support various payment methods. Each of them is configured on the App side to use a different strategy. The strategy defines where and how the Payment Method or the Payment Service Provider elements (mostly the payment form) are displayed in the shop.
+
+The following strategies are available:
+
+- **hosted-payment-page**: The customer is redirected to the PSP page (Hosted Payment Page) after the order is submitted to complete the payment.
+- **express-checkout**: One or more buttons are displayed e.g. on the Cart Page or the PDP Page.
+
+[//]: # (- **embedded**: The payment form is embedded in the shops summary page.)
+
+The strategy is defined in the PaymentMethodTransfer object. The PaymentMethodTransfer object is used to transfer the payment method data between the shop and the PSP. When the App is configured in the ACP App Catalog, the PaymentMethod data is sent via an AsyncAPI to SCOS and is persisted in the database. The configuration is a JSON string which will be mapped to the PaymentMethodTransfer object. 
```

Comment:
What configuration are you referring to?

Reply:
To `When the App is configured in the ACP App Catalog`

Reply:
Is this configuration also sent through the async api?

Reply:
Yes, the configuration is sent from the App to SCOS as a JSON string in the AddPaymentMethod or UpdatePaymentMethod messages.